build/package.sh

```diff
@@ -66,5 +71,5 @@ sed                                \
     -e "s|ARG_ARCH|${ARCH}|g"      \
     -e "s|ARG_SOURCE_BIN|${SOURCE_BIN}|g" \
     Dockerfile.in > .dockerfile-${ARCH}
-docker buildx build --push --pull --sbom=true ${baseimagearg:-} --build-arg kanister_version=${VERSION} -t ${IMAGE}:${VERSION} -f .dockerfile-${ARCH} .
+docker buildx build --push --pull --sbom=true ${baseimagearg:-} --build-arg kanister_version=${VERSION} -t ${IMAGE}:${VERSION} --platform linux/${ARCH}  -f .dockerfile-${ARCH} .
```

Comment:
Note for @PrasadG193: We should make the SBOM flag configurable. `--sbom={ENABLE_SBOM}` or something like that. The packaging will fail if someone doesn't have the correct buildkit enabled.